.gitsplit.yml

```diff
@@ -0,0 +1,15 @@
+# See https://github.com/jderusse/docker-gitsplit
+
+# Path to a cache directory Used to speed up the split over time by reusing git's objects
+cache_url: "/cache/gitsplit"
+
+splits:
+- prefix: "src/Core"
+  target: "https://${GH_TOKEN}@github.com/acmephp/core.git"
+- prefix: "src/Ssl"
+  target: "https://${GH_TOKEN}@github.com/acmephp/ssl.git"
+
+origins:
+- ^master$
+- ^auto-split$
```

Comment:
Only for test, to remove before merge

Reply:
it works => 
Build autoput => https://www.travis-ci.org/acmephp/acmephp/jobs/399009796
New branch on Core => https://github.com/acmephp/core/tree/auto-split